opensrp-chw/src/crvs/assets/json.form/child_enrollment.json

```diff
@@ -328,6 +357,397 @@
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "mother_highest_edu_level",
+        "openmrs_data_type": "select one",
+        "type": "spinner",
+        "hint": "{{child_enrollment.step1.mother_highest_edu_level.hint}}",
+        "values": [
+          "{{child_enrollment.step1.mother_highest_edu_level.value[0]}}",
+          "{{child_enrollment.step1.mother_highest_edu_level.value[1]}}",
+          "{{child_enrollment.step1.mother_highest_edu_level.value[2]}}",
+          "{{child_enrollment.step1.mother_highest_edu_level.value[3]}}",
+          "{{child_enrollment.step1.mother_highest_edu_level.value[4]}}"
+        ],
+        "keys": [
+          "None",
+          "Literacy",
+          "Primary",
+          "Secondary",
+          "University"
+        ],
+        "openmrs_choice_ids": {
+          "None": "none",
+          "Literacy": "",
+          "Primary": "primary",
+          "Secondary": "secondary",
+          "University": "university"
+        }
+      },
+      {
+        "key": "mother_usual_residence",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "motherUsualResidence",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.mother_usual_residence.hint}}",
+        "edit_type": "name",
+        "v_required": {
+          "value": "true",
+          "err": "{{child_enrollment.step1.mother_usual_residence.error}}"
+        }
+      },
+      {
+        "key": "mother_birth_place",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "motherBirthPlace",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.mother_birth_place.hint}}",
+        "edit_type": "name",
+        "v_required": {
+          "value": "true",
+          "err": "{{child_enrollment.step1.mother_birth_place.error}}"
+        }
+      },
+      {
+        "key": "father_name",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "fatherName",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.father_name.hint}}",
+        "edit_type": "name"
+      },
+      {
+        "key": "father_id",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "fatherId",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.father_id.hint}}",
+        "edit_type": "name"
+      },
+      {
+        "key": "father_dob_entered",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "fatherBirthDate",
+        "type": "date_picker",
+        "hint": "{{child_enrollment.step1.father_dob_entered.hint}}",
+        "expanded": false,
+        "duration": {
+          "label": "Age"
+        },
+        "min_date": "today-120y",
+        "max_date": "today-10y"
+      },
+      {
+        "key": "father_dob_unknown",
```

Comment:
There's no relevance attached to either show the date picker or dob unknown checkbox There's also no father_age_entered field that's supposed to show when the checkbox is selected

Reply:
Relevance is added